hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -1098,4 +1098,11 @@ private Constants() {
    */
   public static final String AWS_S3_CENTRAL_REGION = "us-east-1";
 
+  /**
+   * AccessPoint ARN for the bucket. When set globally, all requests to S3 are going through
+   * the AccessPoint pointed to by the ARN. If set as a bucket override then only the requests for
+   * that bucket will go through the AccessPoint. Default value " ".
```

Comment:
default is "", not " " ?

Prefer a separate default const, and for this constant add a @value so any javadocs show it